kong/pdk/log.lua

```diff
@@ -816,7 +816,12 @@ do
       -- seperated by comma or grouped by colon, according to
       -- the nginx doc: http://nginx.org/en/docs/http/ngx_http_upstream_module.html#upstream_status
       local upstream_status = var.upstream_status or ""
-
+      local kong_source = okong.response.get_source(ongx.ctx)
+      if kong_source == "exit" or kong_source == "error" then
```

Comment:
`kong_source` doesn't strike me as a very descriptive name for this; what about `response_source`? 
also, let's wrap the new chunk of code with line breaks to preserve the formatting

Reply:
`response_source` Sounds good to me.